code/game/objects/items/weapons/clown_items.dm

```diff
@@ -11,7 +11,7 @@
 /obj/item/weapon/bananapeel/Crossed(AM as mob|obj)
 	if (istype(AM, /mob/living/carbon))
 		var/mob/M =	AM
-		if(M.lying) return
+		if(M.lying || M.flying) return

```

Comment:
Rather than repeating this code over and over, it might be wiser to make M.walking() proc and check that.